plugins/qlcplus_4.12.1/import.js

```diff
@@ -415,6 +432,177 @@ const qlcPlusChannelPresets = {
   })
 };
 
+const getShutterStrobeCap = (shutterEffect, speedStart, speedEnd, randomTiming) => {
+  const cap = {
+    type: `ShutterStrobe`,
+    shutterEffect
+  };
+
+  if (speedEnd) {
+    cap.speedStart = speedStart;
+    cap.speedEnd = speedEnd;
+  }
+  else if (speedStart) {
+    cap.speed = speedStart;
+  }
+
+  if (randomTiming) {
+    cap.randomTiming = true;
+  }
+
+  return cap;
+};
+
+const getRotationCapType = ({ channelName, channelType }) => {
+  if ([`Pan`, `Tilt`].includes(channelType)) {
+    return `${channelType}Continuous`;
+  }
+
+  if ([`Colour`, `Gobo`].includes(channelType) || /\bgobo\b/i.test(channelName)) {
+    return /wheel\b/i.test(channelName) ? `WheelRotation` : `WheelSlotRotation`;
+  }
+
+  if (channelType === `Prism`) {
+    return `PrismRotation`;
+  }
+
+  return `Rotation`;
+};
+
+const getRotationSpeedCap = (capData, speedStart, speedEnd) => {
+  const cap = {
+    type: getRotationCapType(capData)
+  };
+
+  if (speedEnd) {
+    cap.speedStart = speedStart;
+    cap.speedEnd = speedEnd;
+  }
+  else if (speedStart) {
+    cap.speed = speedStart;
+  }
+
+  return cap;
+};
+
+const qlcPlusCapabilityPresets = {
+  SlowToFast: () => ({
+    type: `Speed`,
+    speedStart: `slow`,
+    speedEnd: `fast`
+  }),
+  FastToSlow: () => ({
+    type: `Speed`,
+    speedStart: `fast`,
+    speedEnd: `slow`
+  }),
+  NearToFar: () => ({
+    type: `Focus`,
+    distanceStart: `near`,
+    distanceEnd: `far`
+  }),
+  FarToNear: () => ({
+    type: `Focus`,
+    distanceStart: `far`,
+    distanceEnd: `near`
+  }),
+  BigToSmall: () => ({
+    type: `Iris`,
```

Comment:
Couldn't this also be `Zoom` if channel group is `Beam`? (Iris' corresponding channel group would be `Shutter`)